src/pages/home/report/ReportActionsView.js

```diff
@@ -95,6 +99,7 @@ const defaultProps = {
     isLoadingReportActions: false,
     isLoadingReportData: false,
     personalDetails: {},
+    network: {},
```

Comment:
With the exception of [IOUConfirmationList.js](https://github.com/Expensify/App/pull/8766/files#diff-6355317ada5b15f6ec56efdb1bbb1401a32afd871213e6bb9a41a75a377f8bc2), the default prop for network is `network: {isOffline: false},`, so I'm curious why this one differs. Is there a reason for this? 

Is it purely because the conditional check differs, and accounts for a null `isOffline` value?

Reply:
Okay, I see other examples so I'm pretty sure this ^ is the explanation.

Reply:
Gonna do a quick test to verify that assumption about `network` always having a value is correct and then will update all these usages.

Reply:
Ok, it's not possible for `network` to not exist as far as I can tell. Updated everything to be `isRequired`, removed `defaultProps`, and made sure we are using `withNetwork()` (not necessary really, but saves us a few Onyx subscriptions).

Reply:
Nice, thanks for explaining too.